server/resource/userLogin.py

```diff
@@ -0,0 +1,34 @@
+from flask_restful import Resource, reqparse
+from flask_jwt_extended import (create_access_token)
+import sys
+sys.path.append("..")
+from models import UserModel
+import datetime
+
+class UserLogin(Resource):
+
+    def post(self):
+        # parse request params, then lookup by email, if email doesnt exist return error message,
+        # if user exists then check password, if password match then return user logged in message,
+        # other wise return error
+        parser = reqparse.RequestParser()
+        parser.add_argument('email',
+                            help='This field cannot be blank', required=True)
+        parser.add_argument('password',
+                            help='This field cannot be blank', required=True)
+        data = parser.parse_args()
+        current_email = UserModel.find_by_email(data['email'])
+
+        if not current_email:
+            return {'message': 'email {} doesn\'t exist'.format(data['email'])}
```

Comment:
we shouldn't tell users that the email doesn't exist (security issue as people can start figuring out who exists in our app) - instead just always return `{'message': 'Wrong credentials'}`